go/cmd/vtclient/vtclient_test.go

```diff
@@ -101,22 +113,45 @@ func TestVtclient(t *testing.T) {
 		{
 			args: []string{"-target", "@master",
 				"SELECT * FROM table1"},
-			rowsAffected: 1,
+			rowsAffected:     1,
+			verifyFunction:   testRowPresence,
+			valuesToVerifies: []int{1, 101},
+		},
+		{
+			args: []string{"-target", "@master", "-bind_variables", `[1]`,
+				"SELECT * FROM table1 where id = :v1"},
+			rowsAffected:     1,
+			verifyFunction:   testRowPresence,
+			valuesToVerifies: []int{1, 101},
 		},
 		{
 			args: []string{"-target", "@master", "-bind_variables", `[ 1 ]`,
 				"DELETE FROM table1 WHERE id = :v1"},
-			rowsAffected: 1,
+			rowsAffected:     1,
+			verifyFunction:   testNoRowPresent,
+			valuesToVerifies: []int{},
 		},
 		{
 			args: []string{"-target", "@master",
 				"SELECT * FROM table1"},
-			rowsAffected: 0,
+			rowsAffected:     0,
+			verifyFunction:   testNoRowPresent,
+			valuesToVerifies: []int{},
 		},
 		{
 			args:   []string{"SELECT * FROM nonexistent"},
 			errMsg: "table nonexistent not found",
 		},
+		{
+			args: []string{"-target", "@master",
+				"SELECT i FROM table1 where id = :v1", "-bind_variables", `[1]`},
```

Comment:
It is convention to use a,b,c etc. as column names, so it will be nice to change this to `SELECT a`